cobra/core/model.py

```diff
@@ -491,32 +487,30 @@ def add_boundary(self, metabolite, type="exchange", reaction_id=None,
         return rxn
 
     def add_reactions(self, reaction_list):
-        """Will add a cobra.Reaction object to the model, if
-        reaction.id is not in self.reactions.
+        """Add reactions to the model.
+
+        Reactions with identifiers identical to a reaction already in the
+        model are ignored.
 
         The change is reverted upon exit when using the model as a context.
 
         Parameters
         ----------
         reaction_list : list
             A list of `cobra.Reaction` objects
-
         """
 
         try:
             reaction_list = DictList(reaction_list)
         except TypeError:
             reaction_list = DictList([reaction_list])
 
-        # Only add the reaction if one with the same ID is not already
-        # present in the model.
-        reactions_in_model = [i.id for i in reaction_list if i.id
-                              in self.reactions]
-
-        if len(reactions_in_model) > 0:
-            raise ValueError(
-                "The following reactions are already in the model: {}"
-                "".format(", ".join(reactions_in_model)))
+        # First check whether the metabolites exist in the model
+        existing = [rxn for rxn in reaction_list if rxn.id in self.reactions]
+        for rxn in existing:
+            LOGGER.info('skip adding reaction %s as already existing' % rxn.id)
```

Comment:
Just nit-picking. It's better to write it with comma instead of the format operator. The way the current line is written, the string is always evaluated thus adding a slight cost even if the logging level ignores info. If you write `.info("... %s ...", rxn.id)` instead, the string will only be formatted if info level is actually desired.